src/Client/Util.java

```diff
@@ -770,8 +770,8 @@ public static String execCmd(String[] cmdArray) throws java.io.IOException {
   }
 
   public static boolean detectBinaryAvailable(String binaryName, String reason) {
-    if (System.getProperty("os.name").contains("Windows")) {
-      return false; // don't trust Windows to run the detection code
+    if (Util.isWindowsOS() || Util.isMacOS()) {
+      return false; // don't trust Windows or macOS to run the detection code
```

Comment:
Actually this code is tested to work on Mac as well. I think it's used to detect if a notify-send binary is available on Mac, which is possible & was a considered use-case at the time of writing that line.

Reply:
Maybe at one point it did but not any more. You can type in anything to the method and it returns true on a Mac. I tested that before making this change.

Reply:
This is actually a huge bug on macOS because it breaks notifications with default settings

Reply:
This actually fixes a huge bug on macOS because it breaks notifications with default settings. I tested on both M1 and Intel macs.

Reply:
Update: took a deeper look and noticed it's just the parsing that's broken, not the actual functionality. Will fix soon.

Reply:
Fixed the parsing... a single extra space character was to blame